projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -188,7 +188,7 @@ public function add_upgrades_menu( $plan = null ) {
 			add_menu_page( __( 'Upgrades', 'jetpack' ), $site_upgrades, 'manage_options', 'paid-upgrades.php', null, 'dashicons-cart', 4 );
 		}
 
-		add_submenu_page( 'paid-upgrades.php', __( 'Plans', 'jetpack' ), __( 'Plans', 'jetpack' ), 'manage_options', 'https://wordpress.com/plans/my-plan/' . $this->domain, null, 1 );
+		add_submenu_page( 'paid-upgrades.php', __( 'Plans', 'jetpack' ), __( 'Plans', 'jetpack' ), 'manage_options', 'https://wordpress.com/plans/' . $this->domain, null, 1 );
```

Comment:
@leogermani I'm afraid this is a bit over my head. Would this be the way to do it?

```suggestion
		add_submenu_page( 'paid-upgrades.php', __( 'Plans', 'jetpack' ), __( 'Plans', 'jetpack' ), 'manage_options', esc_url( Redirect::get_url( 'calypso-backups' ) . $this->domain ), null, 1 );
```

Reply:
Replied in p1624380899326900-slack-CBG1CP4EN

Reply:
Updated in https://github.com/Automattic/jetpack/pull/20141/commits/b5943cb49d07c1973f3e1c3315d61b11f8f90af0 — thanks for all the guidance, @leogermani !